src/components/HighTable/HighTable.tsx

```diff
@@ -262,7 +273,9 @@ export function HighTableInner({
         // we use the scrollRef client width, because we're interested in the content area
         const tableWidth = getClientWidth(scrollRef.current)
         const leftColumnWidth = getOffsetWidth(tableCornerRef.current)
-        setAvailableWidth?.(tableWidth - leftColumnWidth)
+        const available = tableWidth - leftColumnWidth
+        setAvailableWidth?.(available)
+        setAvailableScrollerWidth(available)
```

Comment:
I understand that we don't need the useEffect above, but:


```suggestion
            setScrollNeedInputs?.({ columnIndexes: visibleColumnIndexes, availableWidth: available, waitMs: 50 })
```

Reply:
and remove the `availableScrollerWidth` state